### ISSUE TITLE
Connect WebView Come in after choosing user permission "Allow"

### ISSUE THREAD

**Issue by htarwara6245** [telenordigital/connect-ios-sdk#20]
Please reply me as soon as u see that because its emergency.When I log in to connect using the release version on App Store,it turn back to Connect Web View after choosing user permission (Allow).

Here is the flow -> 
Login-> Web View Appear -> Automatically Log in on Telenor Mobile Data -> Ask user permission that it will use phone number,email,address-> Click Allow->Webview dismiss -> and Show Web View of login page again.

Any help?

**Comment by htarwara6245** [comment]
Here is the log that appear on console when that happen

```
AccessToken not found
RefreshToken not found
tokens not saved
tokens not saved
tokens not saved
tokens not saved
AccessToken not found
RefreshToken not found
```

**Comment by htarwara6245** [comment]
I wonder how this worked?
[http://puu.sh/pyn5J/5e531f87bb.png](url)

Why it return "tokens not saved"? on release version.On build version before releasing to AppStore,everything worked fine.

Any Help?Its already at AppStore.
Here : [Telenor Myanmar - Self Care iOS App](https://itunes.apple.com/us/app/telenor-myanmar-self-care-app/id952597596?mt=8)

If you need to test,Here is demo account for you.Start using "Switch to email"

Email : htarwara.6245@gmail.com
Password : 12345Thiha

**Comment by jorunfa** [comment]
Hi. I'll look at this immediately.

**Comment by jorunfa** [comment]
Found the source of the error: It would fail to save to KeyChain on all devices without passcode becouse of the security settings. Fixed in version 0.6.5: https://github.com/telenordigital/connect-ios-sdk/releases.
